test/POD/burgers.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+"""
+Fine-scale viscous Burgers equation solver.
+
+The equation is
+
+.. math: \frac{\partial u}{\partial t} + \nabla \cdot (\frac{1}{2} u^2 - \epsilon \nabla u) = 0
+"""
+from burgers_init import *
+
+physics.name = "burgers_3d"
+so.name = physics.name
+Profiling.logLevel=5
+Profiling.verbose=True
+ns = NumericalSolution.NS_base(so,[physics],[numerics],so.sList,opts)
```

Comment:
@mfarthin, here I should be able to add the proper simFlags to archive the full nonlinear residual right?